compiler/res/prog8lib/c64/textio.p8

```diff
@@ -556,30 +611,9 @@ _mod		lda  $ffff		; modified
 
 sub  setcc  (ubyte column, ubyte row, ubyte char, ubyte charcolor)  {
 	; ---- set char+color at the given position on the screen
-	%asm {{
-		lda  row
-		asl  a
-		tay
-		lda  setchr._screenrows+1,y
-		sta  _charmod+2
-		adc  #$d4
-		sta  _colormod+2
-		lda  setchr._screenrows,y
-		clc
-		adc  column
-		sta  _charmod+1
-		sta  _colormod+1
-		bcc  +
-		inc  _charmod+2
-		inc  _colormod+2
-+		lda  char
-_charmod	sta  $ffff		; modified
-		lda  charcolor
-_colormod	sta  $ffff		; modified
-		rts
-	}}
+    setchr( column , row, char )
```

Comment:
There's a problem with this refactoring, the routine now has to calculate the memory offsets twice, which slows down the routine. Is this acceptable...
I guess if we don't do this your other modifications don't work

Reply:
yes calculate twice, when use diffrerent banks (1-4) or differente memory location for screen character pointer.

Reply:
maybe better to keep the original setcc, for more speed and add 2 subroutines that calculate the offset of the screen if it is in different position.